lib/utils/forkedblockchain.js

```diff
@@ -161,7 +145,8 @@ ForkedBlockchain.prototype.isFallbackBlock = function(value, callback) {
 };
 
 ForkedBlockchain.prototype.isBlockHash = function(value) {
-  return typeof value === "string" && value.indexOf("0x") === 0 && value.length > 42;
+  const isHash = typeof value === "string" && value.indexOf("0x") === 0 && value.length > 42;
+  return isHash || (Buffer.isBuffer(value) && value.byteLength > 20);
```

Comment:
Reason for change: Sometimes the value is a Buffer and we obviously should check the buffer to see if it big enough to be a blockhash.